static/src/js/containers/AuthCallbackContainer/AuthCallbackContainer.js

```diff
@@ -39,42 +42,40 @@ export const AuthCallbackContainer = ({
       redirect = '/'
     } = params
 
-    // Verify that the redirect params are real URLs
-    try {
-      let redirectUrl
-      if (eddRedirect) redirectUrl = new URL(eddRedirect)
-      if (redirect && redirect !== '/') redirectUrl = new URL(redirect)
-
-      if (
-        redirectUrl
-        && redirectUrl.protocol !== 'http:'
-        && redirectUrl.protocol !== 'https:'
-        && redirectUrl.protocol !== 'earthdata-download:'
-      ) {
-        // The redirectUrl is not a valid protocol
-        console.log('The redirectUrl is not a valid protocol')
-        window.location.replace('/')
+    let eddRedirectUrl = eddRedirect
+    if (redirect.includes('earthdata-download')) {
+      eddRedirectUrl = redirect
+    }
+
+    // Handle EDD redirects
+    if (eddRedirectUrl) {
+      const validEddRedirect = eddRedirectUrl.startsWith('earthdata-download')
```

Comment:
It might be better to do an exact match on the url just in case we use "earthdata-download" in a url sometime down the line. Seems safer to be a little more specific.

Reply:
A little confused about the exact matching. We wouldn't know what the `eddRedirectUrl` value would be only its prefix?

Reply:
Yea so the `fileId` coming from `EDD` is `The authUrl will have a `fileId` parameter appended to it, which is the file ID of the file that triggered the authentication workflow`